resources/roca_lib.brs

```diff
@@ -97,6 +98,16 @@ sub __xit(description as string, func as object)
     m.__suite.__registerCase("skip", description, m.__suite, func)
 end sub
 
+' Fields to add to `m` in the case context.
+' @param fields a roAssociativeObject
+sub __roca_addFields(fields as object)
+    if type(fields) <> "roAssociativeArray" then
+        print "[roca.brs] Error: addFields only accepts a 'roAssociativeArray' - got '" type(fields) "'"
+    else
+        m.__suite.__fields = fields
```

Comment:
This seems to mostly be a place for contextual data — would `m.__suite.__context` (or `.__ctx`) make more sense?

Reply:
Maybe indeed it could be `addContext()` and stored in a `__ctx`.
I'm not sure how to cascade it in sub suites though.

Reply:
Renamed, and documented in readme.